src/main/scala/beam/router/r5/R5Wrapper.scala

```diff
@@ -376,9 +380,15 @@ class R5Wrapper(workerParams: R5Parameters, travelTime: TravelTime, travelTimeNo
     val maybeWalkToVehicle: Map[StreetVehicle, Option[EmbodiedBeamLeg]] =
       accessVehicles.map(v => v -> calcRouteToVehicle(v)).toMap
 
+    val vehicleToInt = (streetVehicle: StreetVehicle) =>
+      maybeWalkToVehicle(streetVehicle)
+        .map(leg => leg.beamLeg.duration)
+        .getOrElse(0)
+
+    @SuppressWarnings(Array("UnsafeTraversableMethods"))
     val bestAccessVehiclesByR5Mode: Map[LegMode, StreetVehicle] = accessVehicles
-      .groupBy(_.mode.r5Mode.flatMap(_.left.toOption).getOrElse(LegMode.valueOf("")))
-      .mapValues(vehicles => vehicles.minBy(maybeWalkToVehicle(_).map(leg => leg.beamLeg.duration).getOrElse(0)))
+      .groupBy(_.mode.r5Mode.get.left.get)
```

Comment:
Same with this. We will introduce TryGet error

Reply:
reverted